content/webapp/components/SearchResults/SearchResults.tsx

```diff
@@ -117,7 +117,7 @@ const SearchResults: FunctionComponent<Props> = ({
             title={item.title || ''}
             partNumber={item.partNumber}
             color={item.color}
-            primaryLabels={[{ text: 'Story' }]}
+            primaryLabels={[]}
```

Comment:
Is it worth a comment here explaining why this is an empty list?

```suggestion
            primaryLabels={
              /* We don't show a label on items that haven't been published yet, because
               * we don't know whether they're a story, comic, etc.
               * See https://github.com/wellcomecollection/wellcomecollection.org/pull/7568 */
              []
            }
```